shippable.yml

```
@@ -9,24 +9,52 @@ matrix:
     - env: T=none
   include:
     - env: T=sanity/1
+    - env: T=sanity/1 A_REV=2.9
+    - env: T=sanity/1 A_REV=2.10
 
     - env: T=units/2.7/1
     - env: T=units/3.6/1
     - env: T=units/3.7/1
     - env: T=units/3.8/1
     - env: T=units/3.9/1
+    - env: T=units/2.7/1 A_REV=2.9
+    - env: T=units/3.6/1 A_REV=2.9
+    - env: T=units/3.7/1 A_REV=2.9
+    - env: T=units/3.8/1 A_REV=2.9
+    - env: T=units/3.9/1 A_REV=2.9
```

Ansible 2.9 does not support Python 3.9:

```suggestion
    - env: T=units/3.9/1 A_REV=2.9
```